### PR TITLE
add Kotlin SDK docs...

### PR DESCRIPTION
**Pull request by astinz**
### Description

This PR adds documentation for Kaptos, a Kotlin Multiplatform SDK for Aptos Integration.

### Covered topics
- **A quick start:** Step-by-step instructions to get you up and running with Kaptos.
- **Data queries:** How to retrieve and handle blockchain data.
- **Account management:**  Creating and managing accounts on Aptos.
- **Transaction Building, Signing and Submission:** Detailed walkthroughs for constructing, authorizing, and submitting transactions.
- And more ...

### CODE REVIEW

**Review by gregnazario — COMMENTED** [review]
Looks pretty good, I skimmed over it.  @hariria thoughts?

File: apps/nextra/pages/en/build/sdks/kotlin-sdk/_meta.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "fetch-data-via-sdk": {
+    title: "Fetch Data via the Kotlin SDK",
+  },
+  "building-transactions": {
+    title: "Building and Sending Transactions",
+  },
+  account: {
+    title: "Creating and Managing Accounts",
+  },
+};
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/kotlin-sdk/_meta.tsx:17]:
```suggestion
export default {
  "---get-started---": {
    type: "separator",
    title: "Get Started",
  },
  quickstart: {
    title: "Quickstart",
  },
  "---usage---": {
    type: "separator",
    title: "Usage",
  },
  "fetch-data-via-sdk": {
    title: "Fetch Data via the Kotlin SDK",
  },
  "building-transactions": {
    title: "Building and Sending Transactions",
  },
  account: {
    title: "Creating and Managing Accounts",
  },
};
```

File: apps/nextra/pages/en/build/sdks/kotlin-sdk/account.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Creating and Managing Accounts"
+---
+
+## Creating and Managing Accounts
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/kotlin-sdk/account.mdx:5]:
```suggestion
# Creating and Managing Accounts
```

File: apps/nextra/pages/en/build/sdks/kotlin-sdk/building-transactions.mdx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+val executedTransaction = aptos.waitForTransaction(HexInput.fromString(commitedTransaction.expect("Transaction failed").hash))
+```
+
+</Steps>
+
+### Full Kotlin Example
+
+The following is a complete example of how to build and send a transaction to transfer APT:
+
+```kotlin
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/kotlin-sdk/building-transactions.mdx:97]:
```suggestion
```kotlin filename="<FILENAME>
```

File: apps/nextra/pages/en/build/sdks/kotlin-sdk/quickstart.mdx [hunk 4]
```diff
@@ -0,0 +1,148 @@
+---
+title: "Kotlin SDK Quickstart"
+---
+
+import { Callout, Steps } from 'nextra/components'
+
+# Kaptos Quickstart
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/kotlin-sdk/quickstart.mdx:7]:
```suggestion
# Kotlin SDK Quickstart
```

File: apps/nextra/pages/en/build/sdks/ts-sdk/quickstart.mdx [hunk 5]
```diff
@@ -2,60 +2,11 @@
...
 
-import { Callout, Steps, Tabs } from 'nextra/components'
+import { Callout, Steps } from 'nextra/components'
 
 # TypeScript SDK Quickstart
 
-<Callout type="info">
-The complete example code can be found at [Full Quickstart Code](#Full_Quickstart_Code) at the bottom of the page.
-</Callout>
-
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/ts-sdk/quickstart.mdx:12]: The changes on this file seem like mistakes... Can you rebase again and remove these changes?

**Review by hariria — COMMENTED** [review]
Some nits

**Review by gregnazario — APPROVED** [review]

**Pull request merged**